build.gradle

```diff
@@ -1,15 +1,21 @@
+import org.gradle.api.file.DuplicatesStrategy
+
 /*
  * Gradle build file for JGribX
  *
- * Target Gradle version: 7.3.2
+ * Target Gradle version: 7.3.3
  *
  * Reference: https://docs.gradle.org/7.3/userguide/building_java_projects.html
  *
  */
+plugins {
+    id "java"
+    id "maven-publish"
+    id "eclipse"
```

Comment:
Doesn't look like the `eclipse` plugin is ever actually used, I think we're okay to remove it